<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSAEncDecDlg</class>
 <widget class="QDialog" name="RSAEncDecDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSA Encrypt/Decrypt</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="mRunBtn">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="mPriKeyPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="mPriKeyBtn">
       <property name="text">
        <string>Private Key</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="mPubKeyEncryptCheck">
     <property name="text">
      <string>Encrypt with public key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mPriKeyAndCertLabel">
     <property name="text">
      <string>Private key and Certificate</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="mCertPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="mCertBtn">
       <property name="text">
        <string>Certificate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="mVersionTypeCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="mOutputTypeCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="mMethodTypeCombo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mChangeBtn">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="mInputText"/>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mInputStringRadio">
       <property name="text">
        <string>String</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mInputHexRadio">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mInputBase64Radio">
       <property name="text">
        <string>Base64</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Len</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mInputLenText">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="mOutputText"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Property and Method</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output Data</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mOutputLenText">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Len</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mInputStringRadio</tabstop>
  <tabstop>mInputHexRadio</tabstop>
  <tabstop>mInputBase64Radio</tabstop>
  <tabstop>mInputText</tabstop>
  <tabstop>mPubKeyEncryptCheck</tabstop>
  <tabstop>mPriKeyPath</tabstop>
  <tabstop>mPriKeyBtn</tabstop>
  <tabstop>mCertPath</tabstop>
  <tabstop>mCertBtn</tabstop>
  <tabstop>mVersionTypeCombo</tabstop>
  <tabstop>mOutputTypeCombo</tabstop>
  <tabstop>mMethodTypeCombo</tabstop>
  <tabstop>mChangeBtn</tabstop>
  <tabstop>mOutputText</tabstop>
  <tabstop>mRunBtn</tabstop>
  <tabstop>mCloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
